core/services/graphql/loaders/DataLoaderCollection.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace EventEspresso\core\services\graphql\loaders;
+
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+use EventEspresso\core\services\collections\Collection;
+use EventEspresso\core\services\collections\CollectionDetails;
+use EventEspresso\core\services\collections\CollectionDetailsException;
+use EventEspresso\core\services\collections\CollectionInterface;
+use EventEspresso\core\services\collections\CollectionLoader;
+use EventEspresso\core\services\collections\CollectionLoaderException;
+
+/**
+ * Class DataLoaderCollection
+ * SplObjectStorage Collection of EventEspresso\core\services\graphql\GQLDataDomainInterface objects
+ *
+ * @package EventEspresso\core\services\graphql
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class DataLoaderCollection extends Collection
+{
+
+    const NAME = 'espresso_graphql_data_loaders';
+
+    const INTERFACE = 'EventEspresso\core\services\graphql\loaders\GQLDataDomainInterface';
+
+    /**
+     * @var CollectionLoader $loader
+     */
+    protected $loader;
+
+
+    /**
+     * DataLoaderCollection constructor
+     *
+     * @throws InvalidInterfaceException
+     */
+    public function __construct()
+    {
+        parent::__construct(
+            DataLoaderCollection::INTERFACE,
+            DataLoaderCollection::NAME
+        );
+    }
+
+
```

Comment:
so much space here, need to tweak prettier to allow also 2 lines of empty spaces in JS :)